src/test/ij.c

```diff
@@ -3419,6 +3482,71 @@ main( hypre_int argc,
       x = (HYPRE_ParVector) object;
    }
 
+   /* Compute RHS squared norm */
+   if (ij_b)
+   {
+      HYPRE_IJVectorInnerProd(ij_b, ij_b, &b_dot_b);
```

Comment:
Hi @victorapm. I'm getting an issue at this line when the RHS is read in from file. The underlying code here tries to do the inner product on the device, but the data is still on the host, which is throwing an error for me.

Reply:
Hi Wayne, does it happen when using `-rhsfromfile`?

Reply:
Wayne, it should work now. Let me know if you have problems :)

Reply:
Yes, it was with `-rhsfromfile` and is fixed now. Thanks!